backend/core/models/recipe.py

```diff
@@ -60,6 +61,12 @@ class Recipe(CommonInfo, SoftDeleteObject):
         default=None,
         help_text="If a clone, the parent this Recipe was cloned from.",
     )
+    tags = ArrayField(
+        base_field=models.TextField(),
```

Comment:
should we be making sure these are unique?

Reply:
we handle that on the UI